### PR TITLE
 Adding account and attendee ID to CSV file

### PR DESCRIPTION
**Pull request by colleen-baltutis**

### CODE REVIEW

File: js/attendeeContact.js [hunk 1]
```diff
@@ -163,9 +164,10 @@ function incrementBadge() {
 
 
 //Builds the data object
-function attendee(id,name, ticket, activeBadges, badgeName, regStatus, artShowHold, opsHold, nonTransferName)
+function attendee(accountId, attendeeId, name, ticket, activeBadges, badgeName, regStatus, artShowHold, opsHold, nonTransferName)
```
Comment (AricStewart) [inline js/attendeeContact.js:167]: You expect accountId to still be important somewhere? We are saving it but see to do nothing more than print it on a debug console.log...

File: js/popup2.js [hunk 2]
```diff
@@ -161,38 +159,41 @@ function doBadgePrintDownload(attendeeData) {
...
     //var textToWrite = "THIS IS SAMPLE TEXT TO WRITE TO A FILE";
     //var textFileAsBlob = new Blob([textToWrite], {type:'text/plain'});
-    var textHeader = '"Badge Type","Badge File BackGround","Badge Number","Badge Name","Workstation ID","Print Number"';
+    var textHeader = '"Badge Type","Badge File BackGround","Account ID","Attendee ID","Badge Name","Workstation ID","Print Number"';
     var text = new Array();
     text[0] = tix;
     text[1] = background;
-    text[2] = badgeNumText;
-    text[3] = badgeNameText;
+    text[2] = accountId;
```
Comment (AricStewart) [inline js/popup2.js:181]: Is this where it is important?  I thought the format of this CSV file was very ridged. You are adding a field. This ok with the badge printer?

Comment (colleen-baltutis) [reply js/popup2.js:181]: Yeah, we'll have to make corresponding edits to the printing script, but this was something Thomas requested.

Comment (AricStewart) [reply js/popup2.js:181]: ok cool, thanks!

**Pull request merged**